app/views/shared/_post.html.erb

```diff
@@ -0,0 +1,5 @@
+<div id="post" class="border-solid border-4 border-black md:w-1/2 mx-auto w-full mb-5">
+  <h2 class="font-bold ml-3"><%= post.title %></h2>
```

Comment:
- [ ] The post on the post index page should be a link to the post show page. Kindly add this link.